src/clr-angular/utils/drag-and-drop/droppable/droppable.ts

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Directive, ElementRef, EventEmitter, Input, OnDestroy, OnInit, Output, Renderer2} from "@angular/core";
+import {Subscription} from "rxjs/Subscription";
```

Comment:
Just import from`rxjs`

Reply:
I believe that change is valid in Angular 6. But this topic branch is still running in Angular 5. Once I rebase to the latest master, I will make this suggested adjustments.

Reply:
Ok, you should get build errors off master, unless you add the rxjs-compat library (please dont).